tests/scripts/thread-cert/border_router/test_multi_ail.py

```diff
@@ -38,7 +38,126 @@
+        br1_1: OtbrNode = self.nodes[self.BR1_1]
+        br2_0: OtbrNode = self.nodes[self.BR2_0]
+        br2_1: OtbrNode = self.nodes[self.BR2_1]
+        br2_2: OtbrNode = self.nodes[self.BR2_2]
+        br2_3: OtbrNode = self.nodes[self.BR2_3]
+
+        border_routers = [br0_0, br1_0, br1_1, br2_0, br2_1, br2_2, br2_3]
+
+        for br in border_routers:
+            br.start()
+
+        # sleep some time more to ensure every BR gets the network data, otherwise the test can fail sometimes.
+        self.simulator.go(config.BORDER_ROUTER_STARTUP_DELAY + 3)
+
+        # ---------------------------------------
+        # br peers
+        _, br1_0_rloc16, br1_1_rloc16, br2_0_rloc16, br2_1_rloc16, br2_2_rloc16, br2_3_rloc16 = [
+            br.get_addr16() for br in border_routers
+        ]
+
+        self.assertCountEqual(br0_0.get_br_peers_rloc16s(), [])
+        self.assertCountEqual(br1_0.get_br_peers_rloc16s(), [br1_1_rloc16])
+        self.assertCountEqual(br1_1.get_br_peers_rloc16s(), [br1_0_rloc16])
+        self.assertCountEqual(br2_0.get_br_peers_rloc16s(), [br2_1_rloc16, br2_2_rloc16, br2_3_rloc16])
+        self.assertCountEqual(br2_1.get_br_peers_rloc16s(), [br2_0_rloc16, br2_2_rloc16, br2_3_rloc16])
+        self.assertCountEqual(br2_2.get_br_peers_rloc16s(), [br2_0_rloc16, br2_1_rloc16, br2_3_rloc16])
+        self.assertCountEqual(br2_3.get_br_peers_rloc16s(), [br2_0_rloc16, br2_1_rloc16, br2_2_rloc16])
+
+        # ---------------------------------------
+        # br routers
+        br0_0_addr = ipaddress.IPv6Address(br0_0.get_ip6_address(config.ADDRESS_TYPE.BACKBONE_LINK_LOCAL))
+        br1_0_addr = ipaddress.IPv6Address(br1_0.get_ip6_address(config.ADDRESS_TYPE.BACKBONE_LINK_LOCAL))
+        br1_1_addr = ipaddress.IPv6Address(br1_1.get_ip6_address(config.ADDRESS_TYPE.BACKBONE_LINK_LOCAL))
+        br2_0_addr = ipaddress.IPv6Address(br2_0.get_ip6_address(config.ADDRESS_TYPE.BACKBONE_LINK_LOCAL))
+        br2_1_addr = ipaddress.IPv6Address(br2_1.get_ip6_address(config.ADDRESS_TYPE.BACKBONE_LINK_LOCAL))
+        br2_2_addr = ipaddress.IPv6Address(br2_2.get_ip6_address(config.ADDRESS_TYPE.BACKBONE_LINK_LOCAL))
+        br2_3_addr = ipaddress.IPv6Address(br2_3.get_ip6_address(config.ADDRESS_TYPE.BACKBONE_LINK_LOCAL))
+
+        self.assertCountEqual(br0_0.get_br_routers_ip_addresses(), [br2_0_addr, br1_0_addr])
+        self.assertCountEqual(br1_0.get_br_routers_ip_addresses(), [br2_0_addr, br0_0_addr])
+        self.assertCountEqual(br1_1.get_br_routers_ip_addresses(), [br2_3_addr])
+        self.assertCountEqual(br2_0.get_br_routers_ip_addresses(), [br0_0_addr, br1_0_addr])
+        self.assertCountEqual(br2_1.get_br_routers_ip_addresses(), [br2_2_addr])
+        self.assertCountEqual(br2_2.get_br_routers_ip_addresses(), [br2_1_addr])
+        self.assertCountEqual(br2_3.get_br_routers_ip_addresses(), [br1_1_addr])
+
+
+@unittest.skip("This is just an example")
+class ExampleTest(thread_cert.TestCase):
```

Comment:
What is the reason to skip this test?

Reply:
Main purpose to save the time to run the full test. This can also be deleted, but want to keep it here as an example case for future development. Any suggestions?